### PR TITLE
Update protobuf source to steam-resources

### PR DESCRIPTION
**Pull request by Crazy-Duck**

### CODE REVIEW

**Comment by Crazy-Duck** [comment]
Fixes #617

File: generate_README.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+jsdoc2md -f index.js handlers/* -t README.hbs > README.md
```
Comment (howardchung) [inline generate_README.sh]: give this file a hashbang?

File: README.hbs [hunk 2]
```diff
@@ -8,13 +8,18 @@ node-dota2
 
 A node-steam plugin for Dota 2, consider it in alpha state.
 
+## Contributing
 Check out @RJacksonm1's blog post (his only blog post), [Extending node-dota2](https://blog.rjackson.me/extending-node-dota2/), for a rough overview of adding new functionality to the library.
 A fair warning, while the way you search for new functionality is still the same, quite a lot has changed (and been simplified) implementation wise.
 It is now easier to implement new functionality than it was back when this blog was written.
 
+Be careful, you'll need SVN if you want to work on the project! Executing an `npm install` inside the project directory
```
Comment (howardchung) [inline README.hbs]: I wonder if we can just pull the protobufs from steamdatabase via curl/wget like we do for the dota2 specific protobufs

File: generate_README.sh (hunk 1 above)
Comment (Crazy-Duck) [reply generate_README.sh]: done

File: README.hbs (hunk 2 above)
Comment (Crazy-Duck) [reply README.hbs]: Well that's what we did before, but with steam moving to steam-resources, might as well follow suit

Comment (Crazy-Duck) [reply README.hbs]: The svn bit stays annoying though, but by bundling them it'll at least not bother users anymore, just contributors.

**Comment by devilesk** [comment]
One thing to note for contributors is that not all the Dota 2 .proto files are currently being loaded by steam-resources. I don't think it's an issue for any existing functionality in node-dota2, but it's a problem if you're trying to add new functionality based on the missing protobufs.

[dota_gcmessages_common_league.proto](https://github.com/SteamDatabase/Protobufs/blob/master/dota2/dota_gcmessages_common_league.proto) is one example I've found that steam-resources doesn't load. The reason is that steam-resources is loading protos from SteamKit's [protos.csv](https://github.com/SteamRE/SteamKit/blob/master/Resources/Protogen/protos.csv) which doesn't include all available proto files.

**Comment by Crazy-Duck** [comment]
@devilesk Any other proto files you'd like to have added? If so I can make a PR to SteamKit to have them added.

**Comment by devilesk** [comment]
dota_gcmessages_common_league.proto is the only one I've needed so far. But here's a list of all the missing protos https://gist.github.com/devilesk/d0d2504962d344bdcde3d10d6d92e669
I think anything starting with dota_gcmessages_ might be of use?

**Pull request merged**